app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -86,19 +95,62 @@ def proof_with_threatmetrix_if_needed(
         end
       end
 
-      def proof_resolution(applicant_pii:, timer:)
+      def proof_residential_address_if_needed(
+        applicant_pii:,
+        timer:,
+        double_address_verification:
+      )
+        return residential_address_unnecessary_result unless double_address_verification
+
+        timer.time('residential address') do
+          resolution_proofer.proof(applicant_pii)
+        end
+      end
+
+      def residential_address_unnecessary_result
+        Proofing::AddressResult.new(
+          success: true, errors: {}, exception: nil, vendor_name: 'ResidentialAddressNotRequired',
+        )
+      end
+
+      def resolution_unnecessary_result
+        Proofing::AddressResult.new(
+          success: true, errors: {}, exception: nil, vendor_name: 'ResolutionUnnecessary',
+        )
+      end
+
+      # TK: confirm vendor name with Team Ada
```

Comment:
I will follow up with Team Ada in Slack about this.

Reply:
I followed up with Team Ada.  They confirmed the decision to 
put the reason for not calling the vendor, rather than putting the name of the vendor.